src/Agent/NewRelic/Agent/Core/Wrapper/OtherTransactionWrapper.cs

```diff
@@ -39,7 +39,8 @@ public AfterWrappedMethodDelegate BeforeWrappedMethod(InstrumentedMethodCall ins
             var trackWorkAsNewTransaction = false;
 
             //If the instrumentation indicates a desire to track this work as a separate transaction, check if this is possible
-            if (instrumentedMethodCall.InstrumentedMethodInfo.RequestedWrapperName == ForceNewTransansactionOnAsyncWrapperName)
+            if (agent.Configuration.ForceNewTransactionOnNewThread ||
```

Comment:
@vuqtran88 Used here :)

Reply:
hmm. This line of code uses the `ForceNewTransactionOnNewThread` configuration instead of the `ForceNewTransactionOnNewThread` attribute.

Reply:
Good catch! I did not mean to include the new attribute in this PR